src/popup/router/pages/Popups/AexPopup.scss

```
@@ -1,4 +1,5 @@
 .popup-aex2 {
+  padding-top: calc(env(safe-area-inset-top));
```

```suggestion
  padding-top: env(safe-area-inset-top);
```
is it a fix to #361?